Rhinoceros_Engine/Create/Point3dList.cs

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
This file should just be called `Point3d.cs` rather than `Point3dList.cs`

Reply:
Huston, we have a problem.
If I do that, I can't keep the name of the method `RandomPoint3dList`, and need to change it to `RandomPoint3d`. 
However, `RandomPoint3d` already exists, and returns `Point3d`, whereas this method would return a `List<Point3d>`. Do you have any suggestion on how to solve this?

Reply:
The method name can remain as it is, as the compliance rule (currently) is:

 - The method return type must match the file name (excluding IEnumerable where the return type is defined as the content of the IEnumerable)

OR

 - The file path must match the return type of the method, AND the method name must match the file name - for more complex cases where you have a thousand ways, for example, to define a `bar` and don't want them all in the same method. You might therefore do a `Bar` folder, and a file called `MyBar.cs` and in there 3 methods called `MyBar` with different inputs.

In the case of this example, I believe you fall happily into the first category, that the method return type `Point3d` can be the file name `Point3d.cs`, even though it is returning a list of the points (the object is still a `Point3d` at its core being returned).

Does that make sense? :smile:

Reply:
It doesn, but I don't think it will work. We have already a method called `Point3d`, which returns a single `Point3d`, rather than a list. 

Maybe I'll go for the second option.

Reply:
I'm not talking about the method though, I'm referring to file name, and return type.

File name is currently `Point3dList.cs` but the return type is `Point3d` - this is the compliance issue. Moving those methods into a `Point3d.cs` file and keep the current, existing method names, is compliant.

Reply:
If you check the commit before the last that's what I did, (https://github.com/BHoM/Rhinoceros_Toolkit/pull/143/commits/4f6ebc668b2b0a4e9a7ddc65469bf684b395ff80) but ci complained again.

Reply:
![image](https://user-images.githubusercontent.com/18049174/75433349-b6a46a80-5947-11ea-8df0-904156a1ca43.png)

According to this the compliance check passed on that commit?